package.json

```diff
@@ -81,7 +81,7 @@
     "@deck.gl/mesh-layers": "^8.8.22",
     "@konturio/default-icons": "^2.2.1",
     "@konturio/default-theme": "3.2.4",
-    "@konturio/ui-kit": "^3.10.1",
+    "@konturio/ui-kit": "^3.11.0",
```

Comment:
just interested, why it's minor version up, not a patch version?

Reply:
@propakov I think because we have a `feat` in the UI
https://github.com/konturio/ui/commit/69a6cd6cdd8b33da7e8d897ad5762e4a435d4c0a